R/test-server.R

```diff
@@ -1,19 +1,22 @@
-#' @noRd
 isModuleServer <- function(x) {
   is.function(x) && names(formals(x))[1] == "id"
 }
 
+isServer <- function(x) {
+  is.function(x) && all(c("input", "output", "session") %in% names(formals(x)))
```

Comment:
Might be good to check that input, output, and session are also the first three arguments.

Reply:
Done